karma.conf.js

```diff
@@ -106,6 +106,7 @@ module.exports = function(config) {
     },
 
     webpackMiddleware: {
+      logLevel: 'error',
```

Comment:
Hm, is this part of the PR? The [default](https://github.com/webpack/webpack-dev-middleware#loglevel) is `'info'`.

Reply:
It's good to have: https://stackoverflow.com/a/53543001/451634